src/util.rs

```diff
@@ -89,3 +117,18 @@ pub fn fzf_helper(now: Epoch, mut dirs: Vec<Dir>) -> Result<Option<String>> {
         _ => bail!("fzf returned an unknown error"),
     }
 }
+
+// FIXME: replace with f64::clamp once it is stable <https://github.com/rust-lang/rust/issues/44095>
+#[must_use = "method returns a new number and does not mutate the original value"]
+#[inline]
+pub fn clamp(val: f64, min: f64, max: f64) -> f64 {
+    assert!(min <= max);
+    let mut x = val;
+    if x < min {
+        x = min;
+    }
+    if x > max {
+        x = max;
+    }
+    x
+}
```

Comment:
Just to make it look nicer.

```suggestion
pub fn clamp(val: f64, min: f64, max: f64) -> f64 {
    assert!(min <= max);
    
    let mut x = val;
    
    if x < min {
        x = min;
    }
    
    if x > max {
        x = max;
    }
    
    x
}
```